### PR TITLE
remove cruft and depend on cluster-config-operator

### PR DESCRIPTION
**Pull request by deads2k**
cluster-config-operator creates our CRDs so they should always be present and we can wait.

/assign @mfojtik

### CODE REVIEW

**Comment by deads2k** [comment]
flaky tests

/retest

File: pkg/operator/configobservation/images/observe_images.go [hunk 1]
```diff
@@ -35,11 +34,6 @@ func ObserveInternalRegistryHostname(genericListers configobserver.Listers, reco
 		}
 	}
 
-	if !listers.ImageConfigSynced() {
```
Comment (mfojtik) [inline pkg/operator/configobservation/images/observe_images.go]: this will make our logs nice again ;-)

Comment (sttts) [reply pkg/operator/configobservation/images/observe_images.go]: why don't we care anymore?

**Pull request merged**